### PR TITLE
update to php versions 5.6.36, 7.0.30, 7.2.5

### PR DESCRIPTION
**Pull request by patricklee2**
RE: http://vstfrd:8080/Azure/RD/_workitems?id=12339075&_a=edit
xdebug no longer supports php5

### CODE REVIEW

File: 7.0.30-apache-xdebug/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM appsvc/php:7.0.6-apache-2
+FROM appsvc/php:7.0.30-apache_1805092223
```
Comment (patricklee2) [inline 7.0.30-apache-xdebug/Dockerfile]: base images currently in appsvctest

File: 7.2.5-apache-xdebug/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM appsvctest/php:7.2.1-apache_1801171635
+FROM appsvc/php:7.2.5-apache_1805092223
```
Comment (patricklee2) [inline 7.2.5-apache-xdebug/Dockerfile:1]: base images currently in appsvctest

File: build.sh [hunk 3]
```diff
@@ -1,26 +1,23 @@
...
 buildnumber=${4-$(date -u +"%y%m%d%H%M")}
 
-# docker build --no-cache -t "$1"/php:5.6.21-apache_"$buildnumber" 5.6.21-apache
-# docker build --no-cache -t "$1"/php:5.6.21-apache-xdebug_"$buildnumber" 5.6.21-apache-xdebug
-# docker build --no-cache -t "$1"/php:7.0.6-apache_"$buildnumber" 7.0.6-apache
-# docker build --no-cache -t "$1"/php:7.0.6-apache-xdebug_"$buildnumber" 7.0.6-apache-xdebug
-# docker build --no-cache -t "$1"/php:7.2.1-apache-xdebug_"$buildnumber" -t "$1"/php:latest-xdebug_"$buildnumber" 7.2.1-apache-xdebug
-docker build --no-cache -t "$1"/php:7.2.1-apache_"$buildnumber" -t "$1"/php:latest_"$buildnumber" 7.2.1-apache
+docker build -q --no-cache -t "$1"/php:5.6.36-apache_"$buildnumber" 5.6.36-apache
+docker build -q --no-cache -t "$1"/php:7.0.30-apache_"$buildnumber" 7.0.30-apache
```
Comment (rramachand21-zz) [inline build.sh:11]: we dont build the xdebug version?

Comment (patricklee2) [reply build.sh:11]: I moved those to build-xdebug.sh

Comment (patricklee2) [reply build.sh:11]: the xdebug images need the new base images to be in appsvc and needs to know the new buildnumber, building both at the same time wont work.

**Pull request merged**